gulp-tasks/release.js

```diff
@@ -77,24 +78,25 @@ gulp.task("zip-archives", (done) => {
   zip.on("close", (code) => {
     if (code === 0) {
       createHash(`dist/${dutil.dirName}.zip`);
+      const stats = fs.statSync(`./dist/${dutil.dirName}.zip`);
```

Comment:
including this here to output zip size, we could port this over into production sooner if we think it is of value